### PR TITLE
Add string functions

### PR DESCRIPTION
**Pull request by frinyvonnick**
**Issue:** #79

- [x] capitalize
- [x] replace
- [x] toLower
- [x] toUpper

### CODE REVIEW

File: src/string/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Converts the first character of string to upper case and the remaining to lower case.
+ * @function
+ * @memberof object
+ * @param {Object} object The object to modify.
+ * @param {Array|string} path The path of the property to set.
+ * @return {Object} Returns the updated object.
+ * @example capitalize({ nested: { a: "a string" } }, 'nested.a') // => { nested: { a: "A string" } }
+ * @see {@link https://lodash.com/docs#capitalize|lodash.capitalize} for more information.
+ * @see {@link https://lodash.com/docs#identity|lodash.identity} for more information.
```
Comment (nlepage) [inline src/string/capitalize.js]: Remove this link

File: src/string/capitalize.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+import { capitalize } from './capitalize'
+import { immutaTest } from 'test.utils'
+
+describe('Capitalize', () => {
+
+  it('should converts the first character of string to upper case and the remaining to lower case', () => {
```
Comment (nlepage) [inline src/string/capitalize.spec.js]: "should convert ..." -s

File: src/string/replace.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @function
+ * @memberof object
+ * @param {Object} object The object to modify.
+ * @param {Array|string} path The path of the property to set.
+ * @param {RegExp|string} pattern The pattern to replace.
+ * @param {Function|string} replacement The match replacement.
+ * @return {Object} Returns the updated object.
+ * @example replace({ nested: { a: "Hi Nico" } }, 'nested.a', 'Nico', 'Yvo') // => { nested: { a: "Hi Yvo" } }
+ * @see {@link https://lodash.com/docs#replace|lodash.replace} for more information.
+ * @see {@link https://lodash.com/docs#identity|lodash.identity} for more information.
```
Comment (nlepage) [inline src/string/replace.js]: Remove this link

File: src/string/toLower.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Converts string, as a whole, to lower case just like String#toLowerCase.
+ * @function
+ * @memberof object
+ * @param {Object} object The object to modify.
+ * @param {Array|string} path The path of the property to set.
+ * @return {Object} Returns the updated object.
+ * @example toLower({ nested: { a: "A STRING" } }, 'nested.a') // => { nested: { a: "a string" } }
+ * @see {@link https://lodash.com/docs#toLower|lodash.toLower} for more information.
+ * @see {@link https://lodash.com/docs#identity|lodash.identity} for more information.
```
Comment (nlepage) [inline src/string/toLower.js]: Remove this link

File: src/string/toLower.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import _toLower from 'lodash/toLower'
+import { convert } from 'util/convert'
+
+/**
+ * Converts string, as a whole, to lower case just like String#toLowerCase.
```
Comment (nlepage) [inline src/string/toLower.js]: Add link to https://mdn.io/String/toLowerCase

File: src/string/toLower.spec.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+import { immutaTest } from 'test.utils'
+import { toLower } from './toLower'
+
+describe('toLower', () => {
+
+  it('should converts string, as a whole, to lower case just like String#toLowerCase', () => {
```
Comment (nlepage) [inline src/string/toLower.spec.js]: "should convert" -s

File: src/string/toUpper.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Converts string, as a whole, to upper case just like String#toUpperCase.
+ * @function
+ * @memberof object
+ * @param {Object} object The object to modify.
+ * @param {Array|string} path The path of the property to set.
+ * @return {Object} Returns the updated object.
+ * @example toUpper({ nested: { a: "a string" } }, 'nested.a') // => { nested: { a: "A STRING" } }
+ * @see {@link https://lodash.com/docs#toUpper|lodash.toUpper} for more information.
+ * @see {@link https://lodash.com/docs#identity|lodash.identity} for more information.
```
Comment (nlepage) [inline src/string/toUpper.js]: Remove this link

File: src/string/toUpper.spec.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+import { immutaTest } from 'test.utils'
+import { toUpper } from './toUpper'
+
+describe('toUpper', () => {
+
+  it('should converts string, as a whole, to upper case just like String#toUpperCase', () => {
```
Comment (nlepage) [inline src/string/toUpper.spec.js]: "should convert" -s

File: src/string/toUpper.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import _toUpper from 'lodash/toUpper'
+import { convert } from 'util/convert'
+
+/**
+ * Converts string, as a whole, to upper case just like String#toUpperCase.
```
Comment (nlepage) [inline src/string/toUpper.js]: Add link to https://mdn.io/String/toUpperCase

File: src/string/index.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+import { toLower } from './toLower'
+import { toUpper } from './toUpper'
+
+/**
+ * String functions.
+ * @namespace string
+ * @since 0.3.0
+ */
+export {
+  capizalize,
```
Comment (charlyx) [inline src/string/index.js]: 👎

Comment (fabienjuif) [reply src/string/index.js]: 🍺

Comment (kefranabg) [reply src/string/index.js]: 💩

**Pull request merged**